pedro-monteiro/1.ii/index.html

```diff
@@ -0,0 +1,31 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <title>Curriculum Vitae</title>
+</head>
+<body>
+    <div align="center">
```

Comment:
Please do not use anymore this attribute `align="center"` as it break the meaning of HTML and also because it is [deprecated](https://www.codehelp.co.uk/html/deprecated.html). Any styling or formatting will be CSS responsibility.